### ISSUE TITLE
Timer interupts

### ISSUE THREAD

**Issue by OllieLollie1** [LdB-ECM/Raspberry-Pi#10]
Hi LdB it's me again. Just wondering what the nCNTPNSIRQ_IRQ register bit is? I'm not sure of its purpose, I just know it is required for the timer interrupt otherwise you wouldn't have it in your code I wouldn't think.

**Comment by OllieLollie1** [comment]
Also  does the interrupt need to be set up for the timer at the very start or can it be later as long as it is after the core3_deadloop

**Comment by OllieLollie1** [comment]
I tried to integrate the core interrupt code with my code and I must have made an error in setting up the interrupt because the interrrupt fails to fire on core 3 witha timer IRQ I can fire an interrupt when it encounters a brk #0 instruction and the print works correctly in dbg_main or debug_decodeexc so my guess is that I have incorrectly set up the interrupt.

The bug you mentioned before on the forums about the debugger not showing up was because I had a hang command in my vector table. I have removed that command for obvious reasons. 

I know this is an IRQ and not a synchronous exception but the handler for each is quite similar apart from the fact one is a console and one flip flops an led.
